cognite_toolkit/cdf_tk/load.py

```diff
@@ -751,26 +803,30 @@ def create(self, items: Sequence[Transformation], drop: bool, filepath: Path) ->
             )
             for dup in e.duplicated:
                 print(f"           {dup.get('externalId', 'N/A')}")
-            return []
+            return TransformationList([])
         except Exception as e:
             print(f"[bold red]ERROR:[/] Failed to create resource(s).\n{e}")
             self.ToolGlobals.failed = True
             return TransformationList([])
-        return created
+        return cast(TransformationList, created)
```

Comment:
Why doesn't TransformationList(created) work here? It seems inconsistent?

Reply:
Bug in the SDK, it does not overload .create properly. I have fixed it in my PR.